src/styles.scss

```diff
@@ -1 +1,6 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+// Ваші власні SCSS стилі можуть йти після директив Tailwind
```

Comment:
let's add comments in English only. This comment looks redundant, the comment below is enough.